massa-ledger/src/ledger.rs

```diff
@@ -106,9 +106,10 @@ impl FinalLedger {
     /// # Arguments
     /// * configuration: ledger configuration
     /// * state: bootstrap state
-    pub fn from_bootstrap_state(config: LedgerConfig, state: FinalLedgerBootstrapState) -> Self {
+    pub fn from_bootstrap_state(config: LedgerConfig, _state: FinalLedgerBootstrapState) -> Self {
+        // dummy implementation since bootstrap streaming is currently developed
         FinalLedger {
-            sorted_ledger: state.sorted_ledger,
+            sorted_ledger: LedgerDB::new(),
```

Comment:
I think we should make a temporary bootstrap state method because otherwise this PR will block launching new testnets and we shouldn't merge something that prevent launching testnets.